### PR TITLE
docs: add BentoCloud docs

### PR DESCRIPTION
**Pull request by jianshen92**
## What does this PR address?
* Adding BentoCloud docs into the main documentation
* Change logo/favicons to new ones
* Modify the section in the navbar
image

## Before submitting:

- [ ] Does the Pull Request follow [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary) naming? Here are [GitHub's
      guide](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) on how to create a pull request.
- [ ] Does the code follow BentoML's code style, `pre-commit run -a` script has passed ([instructions](https://github.com/bentoml/BentoML/blob/main/DEVELOPMENT.md#style-check-auto-formatting-type-checking))?
- [ ] Did you read through [contribution guidelines](https://github.com/bentoml/BentoML/blob/main/CONTRIBUTING.md#ways-to-contribute) and follow [development guidelines](https://github.com/bentoml/BentoML/blob/main/DEVELOPMENT.md#start-developing)?
- [ ] Did your changes require updates to the documentation? Have you updated
      those accordingly? Here are [documentation guidelines](https://github.com/bentoml/BentoML/tree/main/docs) and [tips on writting docs](https://github.com/bentoml/BentoML/tree/main/docs#writing-documentation).
- [ ] Did you write tests to cover your changes?

### CODE REVIEW

File: docs/source/bentocloud/getting-started/build.rst [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+BentoML embraces **open standards** for AI applications and promotes **best practices** to enhance the quality of your work. It offers a versatile framework capable of unifying **online**, **offline**, and **streaming** workloads.
+
+In essence, BentoML aims to simplifies the process of building ML models, ensuring your team can focus on what truly matters: creating AI applications that solve real-world problem.
+
+-------------------
+Building Your Bento
+-------------------
+
+Let's delve into how to **BUILD** a service with HuggingFace’s transformers.
```
Comment (ssheng) [inline docs/source/bentocloud/getting-started/build.rst:23]:
```suggestion
Let's delve into how to **BUILD** an NLP application that categorizes and summarizes text with HuggingFace’s Transformers. We will be demonstrating with [Transformers NLP Service](https://github.com/bentoml/transformers-nlp-service) project.
```

File: docs/source/bentocloud/getting-started/build.rst [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+To learn how to deploy your Bento with BentoCloud, read :doc:`Getting Started -- Ship To Production <ship>`
+
+.. _bento-gallery:
+
+-------------
+Bento Gallery
+-------------
+
+The Bento Gallery is a curated collection showcasing various types of ML models built and served using BentoML. Explore, learn, and draw inspiration from these showcased projects.
```
Comment (ssheng) [inline docs/source/bentocloud/getting-started/build.rst:55]:
```suggestion
The [Bento Gallery](https://bentoml.com/gallery) is a curated collection showcasing various types of ML models built and served using BentoML. Explore, learn, and draw inspiration from these showcased projects.
```

File: docs/source/bentocloud/getting-started/ship.rst [hunk 3]
```diff
@@ -0,0 +1,153 @@
+==================
+Ship to Production
+==================
+
+BentoCloud offers a full suite of MLOps lifecycle management tools right off the bat, enabling developers and ML practitioners to zero in on **BUILDING** while we take care of the intricate infrastructure work - **WE SHIP**.
```
Comment (ssheng) [inline docs/source/bentocloud/getting-started/ship.rst:5]:
```suggestion
BentoCloud offers a full suite of MLOps lifecycle management tools right off the bat, enabling developers and ML practitioners to zero in on **BUILDING** while we take care of the intricate infrastructure work.
```

File: docs/source/bentocloud/getting-started/ship.rst [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+
+The topics covered in this article are:
+
+- **Deploy** your Bento into production within minutes
+- Manage Versioning and Revisions for your Bento and Deployment
+- Integrate with popular **MLOps workflow** engines, like Kubeflow and Airflow
+- Activate **GPU usage** with a simple click
+- Utilize out-of-the-box DevOps and ML **Observability**
```
Comment (ssheng) [inline docs/source/bentocloud/getting-started/ship.rst:13]:
```suggestion
- **Deploy** your Bento into production within minutes
- Manage application versioning and deployment revisions
- Integrate with popular **MLOps** ecosystem tooling
- Activate **GPU** acceleration with a simple click
- Utilize out-of-the-box DevOps and ML **Observability**
```

File: docs/source/bentocloud/getting-started/ship.rst [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+.. code-block:: bash
+
+    bentoml yatai login --api-token <your-api-token> --endpoint <https://cloud.bentoml.com>
+
+If the login is successful, you should see the following output:
+
+.. code-block:: bash
+
+    Successfully logged in as user "user" in organization "mybentocloud".
+
```
Comment (ssheng) [inline docs/source/bentocloud/getting-started/ship.rst:43]:
```suggestion

.. note::
    BentoCloud organizations function as collaborative workspaces where your team can effectively manage models and bentos. Additionally, these organizations facilitate the management and monitoring of active deployments, enabling you to track the creations of your team efficiently.

```

File: docs/source/bentocloud/getting-started/ship.rst [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+After successfully pushing your Bento, navigate to `<https://cloud.bentoml.com/bento_repositories>`_ and you'll find your Bento deployed in BentoCloud!
+
+.. image:: ../../_static/img/bentocloud/ship-pushed-bento.png
+
+--------------------
+Deploying your Bento
+--------------------
+
+Bento can be deployed in two ways: as an Online Service or as a Serverless Function. The choice depends on your specific use-case. You can learn more about the implications and best scenarios for each method in `Deploying Bento <https://www.notion.so/Deploying-Bento-6b38db89b1bf4319adbb6d33a8a30e8f?pvs=21>`_.
```
Comment (ssheng) [inline docs/source/bentocloud/getting-started/ship.rst:73]: This link needs an update. @jianshen92

File: docs/source/bentocloud/getting-started/ship.rst [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+.. image:: ../../_static/img/bentocloud/ship-deploy-bento.png
+
+Please follow these steps:
+
+1. Select **Basic** on the upper left.
+2. Choose the appropriate **Cluster**.
+3. Give your **deployment** a unique name, for instance, ``my-first-deployment``.
+4. Enable **Public Access**.
+5. Choose the desired **Bento Repository**.
+6. Select the correct **Bento Version**.
```
Comment (ssheng) [inline docs/source/bentocloud/getting-started/ship.rst:86]:
```suggestion
1. Choose the appropriate **Cluster**.
2. Give your **deployment** a unique name, for instance, ``my-first-deployment``.
3. Choose the desired **Bento Repository** and select the correct **Bento Version**.
```

File: docs/source/bentocloud/getting-started/ship.rst [hunk 8]
```diff
@@ -0,0 +1,153 @@
...
+
+    curl -X 'POST' \
+      'http://<your-deployment-url>/summarize' \
+      -H 'accept: text/plain' \
+      -H 'Content-Type: text/plain' \
+      -d 'Your Text Here'
+
+---------------------
+Collaborative Scaling
+---------------------
```
Comment (ssheng) [inline docs/source/bentocloud/getting-started/ship.rst:111]:
```suggestion
--------------------
Team Collaboration
--------------------
```

File: docs/source/bentocloud/getting-started/ship.rst [hunk 9]
```diff
@@ -0,0 +1,153 @@
...
+
+Scaling isn't limited to handling larger workloads; it's also about fostering collaboration within and across teams. 
+BentoCloud facilitates this process through its **Model and Bento Repositories**. 
+
+These repositories provide a platform for ML practitioners and engineers to **publish**, **share**, and **distribute** their work with ease. You can also **download pre-built models** from popular repositories such as HuggingFace's model hub.
```
Comment (ssheng) [inline docs/source/bentocloud/getting-started/ship.rst:116]:
```suggestion
These repositories provide a platform for team members to **publish**, **share**, and **distribute** their work with ease. You can also **import pre-trained models** from popular repositories such as HuggingFace's model hub.
```

File: docs/source/bentocloud/getting-started/ship.rst [hunk 10]
```diff
@@ -0,0 +1,153 @@
...
+
+Within the dynamic field of Machine Learning, systems typically evolve in two ways:
+
+1. **Code Changes**:
+Much like traditional software development, engineers modify the code base, implementing bug fixes, new features, and updates to data processing routines.
+
+2. **Data/Model Changes**:
+Without altering any code, Machine Learning practitioners may retrain an existing model with a different dataset. This adjustment can result in the system producing different output behavior.
+
+Both types of changes merit the creation of a newly built Bento. In BentoML, Bentos are **namespaced** with the **name** and **versioned** according to their **tags**, allowing you to track the evolution of your projects over time. You can also assign additional **labels** to each version, providing extra granularity for downstream deployment filtering.
```
Comment (ssheng) [inline docs/source/bentocloud/getting-started/ship.rst:132]:
```suggestion
Both types of changes merit the creation of a newly built Bento. In BentoML, Bentos are **uniquely referenced** with the **name** and **versioned** according to their **tags**, allowing you to track the evolution of your projects over time. You can also assign additional **labels** to each version, providing extra granularity for downstream deployment filtering.
```

File: docs/source/bentocloud/getting-started/ship.rst [hunk 11]
```diff
@@ -0,0 +1,153 @@
...
+
+The same principles apply to **Deployments**. You can iteratively create new revisions of your service with various configurations, such as a new Bento Version or different resource settings. If a new deployment exhibits undesired behavior, you have the flexibility to revert back to a previous, stable version at any time.
```
Comment (ssheng) [inline docs/source/bentocloud/getting-started/ship.rst:134]:
```suggestion
The same principles apply to **Deployments**. You can iteratively create new revisions of your service with various configurations, such as a new bento version or different resource settings. If a new deployment exhibits undesired behavior, you have the flexibility to revert back to a previous, stable version at any time.
```

File: docs/source/bentocloud/getting-started/ship.rst [hunk 12]
```diff
@@ -0,0 +1,153 @@
...
+
+.. image:: ../../_static/img/bentocloud/ship-versioning.png
+
+-------------
+Observability
+-------------
+
+In Machine Learning systems, there are two distinct types of observability: **DevOps Observability** and **ML Observability.** 
+DevOps Observability focuses on the monitoring, logging, and tracing of technical resources. 
+On the other hand, ML Observability concentrates on the tracking of feature and model drift.
```
Comment (ssheng) [inline docs/source/bentocloud/getting-started/ship.rst:144]:
```suggestion
On the BentoCloud platform, you have the capability to monitor the status and health of both deployments and models. For application-level monitoring, the platform provides access to logs, metrics, and traces for each individual deployment, providing you with a comprehensive overview of application performance. For model-level monitoring, you can collect inference data with the :ref:`model monitoring API <bentoml/guides/monitoring:Build an ML Application with monitoring API>` and track model drift over time using a monitoring platform of your choice. 
```

File: docs/source/bentocloud/getting-started/scale.rst [hunk 13]
```diff
@@ -0,0 +1,73 @@
+================
+Scale for Growth
+================
+
+In the previous parts of our 'Getting Started' series:
+
+- :doc:`Getting Started -- Build AI Application <build>`
+- :doc:`Getting Started -- Ship To Production <ship>`
+
+We've shown you how to build your application with BentoML and deploy it using BentoCloud. Now, in this final article, we'll explain how BentoCloud makes scaling your application straightforward. BentoCloud is a fully managed platform that aims to relieve users of hardware concerns, allowing you to pay only for what you use.
```
Comment (ssheng) [inline docs/source/bentocloud/getting-started/scale.rst:10]:
```suggestion
We've shown you how to build your application with BentoML and deploy it using BentoCloud. Now, in this final article, we'll explain how BentoCloud makes scaling your application straightforward. BentoCloud is a fully managed platform that aims to relieve users of infrastructure concerns, allowing you to pay only for what you use.
```

File: docs/source/bentocloud/getting-started/scale.rst [hunk 14]
```diff
@@ -0,0 +1,73 @@
...
+-------------------
+Resource Management
+-------------------
+
+~~~~~~~~~~~~~~~~~~~~~~
+Auto Scaling Instances
+~~~~~~~~~~~~~~~~~~~~~~
+
+BentoCloud is built distributedly, taking full advantage of the auto scaling capabilities provided by its  `Horizontal Pod Autoscaler <https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/>`_ (HPA). 
+We've set up a range of metrics that trigger the autoscaler, while also giving expert users the freedom to override these default scaling policies. This can be done via a JSON editor or directly through the BentoCloud API.
```
Comment (ssheng) [inline docs/source/bentocloud/getting-started/scale.rst:21]:
```suggestion
BentoCloud is designed with scalability at its core. Scaling operations can be performed based on various parameters including CPU or GPU usage, or metrics related to traffic volume. The platform's advanced configuration settings enable a wide range of scaling behaviors, accommodating needs from rapid scaling for high-growth scenarios to stable scaling for consistent demand levels. Importantly, BentoCloud operates on a pay-as-you-go model, ensuring users are only charged for the resources they have actually utilized, promoting cost-efficiency and flexibility.
```

File: docs/source/bentocloud/getting-started/scale.rst [hunk 15]
```diff
@@ -0,0 +1,73 @@
...
+For those seeking advanced scaling behaviour, your services can scale according to `custom metrics <https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#scaling-on-custom-metrics>`_. 
+This feature is exclusively available to users with **Enterprise** plan.
+
+~~~~~~~~~~
+Scale-To-Zero
+~~~~~~~~~~
+
+When you deploy a Bento, you can opt for an on-demand function mode of deployment. 
+This is BentoML's serverless approach, ideally suited for use cases where ML models are large, such as high-resolution image models or large language models. 
+Instances can scale down to zero when not in use, offering a cost-effective solution, especially when using pricier GPU instances.
```
Comment (ssheng) [inline docs/source/bentocloud/getting-started/scale.rst:34]:
```suggestion
BentoCloud supports deploying applications as on-demand functions, enhancing its flexibility and resource efficiency. In this deployment mode, instances can scale down to zero when there's no usage on the API server or runners, minimizing idle resources. To ensure high reliability during a cold start, BentoCloud queues requests, preventing potential overloads. This deployment strategy is particularly advantageous for applications handling a large number of models and sporadic requests, as only those models actively in use are allocated resources. Additionally, this mode of operation provides significant cost efficiency, especially for GPU-accelerated instances, by tailoring resource allocation to real-time demand.
```

**Comment by aarnphm** [comment]
Very nit: can we make the button the size of the whole docs,and it would be cool if we can add backdrop shadow for it

File: docs/source/bentocloud/how-tos/deploy-bentos.rst [hunk 16]
```diff
@@ -0,0 +1,64 @@
...
+
+Deploying a machine learning model can be quite a task. However, BentoCloud simplifies the process by enabling you to serve your model as an online service or on-demand function. This guide will walk you through how to deploy a Bento on BentoCloud.
+
+Online Service vs On-Demand Function
+=====================================
+
+There are two modes of deployment on BentoCloud:
+
+1. **Online Service:** This type of deployment spins up a persistent server that serves your model and is ideal for cases where low latency responses are crucial. A minimum of one running instance is required.
+2. **On-Demand Function:** In this mode, instances are created when requests come in and are shut down when idle. During cold start, requests will be queued while the instance is spinning up. This approach is cost-effective for sporadic workloads, and for larger models like LLMs and Image models, as GPU instances can scale to zero when not in use.
```
Comment (ssheng) [inline docs/source/bentocloud/how-tos/deploy-bentos.rst:13]:
```suggestion
--------------
Online Service
--------------

The online service mode of deployment offered by BentoCloud is ideal for low-latency serving scenarios. To ensure requests can be promptly addressed, instances in this mode are never scaled down to zero, maintaining a ready state for immediate processing. Furthermore, requests are directly routed to the serving instances of the API Server and Runners, bypassing any queuing mechanisms. This direct routing mechanism ensures minimum latency, providing an efficient and swift response to incoming requests.

----------------------
On-Demand Function
----------------------

The on-demand function mode of deployment offered by BentoCloud is particularly suited for situations that prioritize cost-efficiency and reliability. In scenarios where requests are sporadic, this mode enables instances to scale down to zero, thereby conserving resources. This feature proves particularly beneficial for GPU-accelerated instances, which are generally more expensive to maintain. To ensure maximum reliability, especially during periods of cold-start or overload, requests are queued prior to processing. This mechanism enables the system to handle bursts of requests effectively, thus enhancing the robustness and dependability of your application under varying load conditions.

```

File: docs/source/bentocloud/how-tos/deploy-bentos.rst [hunk 17]
```diff
@@ -0,0 +1,74 @@
+================
+Deploy Bentos
+================
+
+Deploying a machine learning model can be quite a task. However, BentoCloud simplifies the process by enabling you to serve your model as an online service or on-demand function. This guide will walk you through how to deploy a Bento on BentoCloud.
+
+Online Service vs On-Demand Function
+=====================================
```
Comment (ssheng) [inline docs/source/bentocloud/how-tos/deploy-bentos.rst:8]:
```suggestion
Deployment Mode
==============
```

File: docs/source/bentocloud/how-tos/index.rst [hunk 18]
```diff
@@ -0,0 +1,22 @@
+=============
+How-to Guides
+=============
+
+How-to guides take the reader through the steps required to solve a problem
```
Comment (ssheng) [inline docs/source/bentocloud/how-tos/index.rst:5]:
```suggestion
How-to guides take the reader through the steps required to solve a problem.
```

File: docs/source/bentocloud/how-tos/manage-access-token.rst [hunk 19]
```diff
@@ -0,0 +1,104 @@
...
+Manage Access Tokens
+======================
+
+This guide will help you understand how to manage access and permissions for
+programmatic interaction with BentoCloud using access tokens. BentoCloud uses
+two types of access tokens:
+
+1. Access tokens for interacting with BentoCloud resources (e.g., clusters,
+   models, Bentos, deployments, user management).
+2. Access tokens for accessing deployed API servers.
```
Comment (ssheng) [inline docs/source/bentocloud/how-tos/manage-access-token.rst:11]:
```suggestion
In BentoCloud, API tokens serve as the means of authorization for two distinct scopes: **BentoCloud resources** and **Bento deployment** authorization. The scope of BentoCloud resources includes authorization for APIs that facilitate the pushing and pulling of models and bentos, along with actions related to creating, updating, and terminating deployments. On the other hand, the deployment scope encapsulates the authorization required for invoking deployed Bento applications.
```

File: docs/source/bentocloud/how-tos/manage-access-token.rst [hunk 20]
```diff
@@ -0,0 +1,104 @@
...
+These tokens are used as part of the Basic Authentication Scheme.
+
+For HTTP-based servers, include the token in the header of your HTTP request like this:
+
+.. code-block:: bash
+
+   curl "http://cloud.bentoml.com" \
+     -H "Content-Type: application/json" \
+     -H "Authorization: Basic $YOUR_TOKEN" \
+     --data '{"prompt": "What state is Los Angeles in?", "max_length": 100}'
```
Comment (ssheng) [inline docs/source/bentocloud/how-tos/manage-access-token.rst:91]:
```suggestion
   curl "http://app-name.organization.cloud-apps.bentoml.com" \
     -H "Content-Type: application/json" \
     -H "Authorization: Basic $YOUR_TOKEN" \
     --data '{"prompt": "What state is Los Angeles in?", "max_length": 100}'
```

File: docs/source/bentocloud/how-tos/manage-access-token.rst [hunk 21]
```diff
@@ -0,0 +1,104 @@
...
+To create a new token:
+
+1. Click the `Create` button in the top-right corner of the page.
+
+.. image:: ../../_static/img/bentocloud/ship-get-api-token.png
+    
+
+1. In the dialog box that appears, select the desired scope for the token
+   and set an expiration date if needed.It's a best practice to always
+   set an expiration date for access tokens.
```
Comment (ssheng) [inline docs/source/bentocloud/how-tos/manage-access-token.rst:37]:
```suggestion
1. In the dialog box that appears, select the desired scope for the token
   and set an expiration date if needed. It's a best practice to always
   set an expiration date for access tokens.
```

File: docs/source/bentocloud/how-tos/manage-access-token.rst [hunk 22]
```diff
@@ -0,0 +1,104 @@
...
+-------------------
+
+You can find the API Tokens tab on each deployment page.
+Here, you can view the metadata associated with each API token. To create a new token:
+
+1. Click the `Create` button in the top-right corner.
+
+Just like with BentoCloud resource tokens, you can specify the scope and
+set an expiration date for each token. Remember, setting an expiration date
+for all access tokens is considered a best practice for security.
```
Comment (ssheng) [inline docs/source/bentocloud/how-tos/manage-access-token.rst:77]:
```suggestion
2. Select the scopes and expiration.
3. Click submit.
```

File: docs/source/bentocloud/how-tos/manage-models-and-bentos.rst [hunk 23]
```diff
@@ -0,0 +1,114 @@
...
+
+   client.list_models()
+
+   client.push_model(model)
+
+   client.push_bento(bento)
+
+   client.pull_model("model-tag")
+
+   client.pull_bento("bento-tag")
```
Comment (ssheng) [inline docs/source/bentocloud/how-tos/manage-models-and-bentos.rst:110]: Let's link the API reference here instead.
```suggestion
```

File: docs/source/bentocloud/topics/observability.rst [hunk 24]
```diff
@@ -0,0 +1,38 @@
+Observability
+================
+
+Machine Learning systems necessitate two types of observability: **DevOps Observability** and **ML Observability**.
```
Comment (ssheng) [inline docs/source/bentocloud/topics/observability.rst:4]:
```suggestion
```

File: docs/source/bentocloud/references/index.rst [hunk 25]
```diff
@@ -0,0 +1,3 @@
+==========
+References
+==========
```
Comment (ssheng) [inline docs/source/bentocloud/references/index.rst:3]:
```suggestion
==========

.. TODO:
  Add API reference.
```

File: docs/source/index.rst [hunk 26]
```diff
@@ -128,15 +129,21 @@ contributing to the project, helping other users and discuss all things MLOps.
...
+
+.. toctree::
+   :caption: BentoCloud
+   :hidden:
+   
+   bentocloud/getting-started/index
+   bentocloud/how-tos/index
+   bentocloud/topics/index
+   bentocloud/tutorials/index
+   bentocloud/reference/index
```
Comment (ssheng) [inline docs/source/index.rst:141]:
```suggestion
   bentocloud/references/index
```

File: docs/source/index.rst [hunk 27]
```diff
@@ -128,15 +129,21 @@ contributing to the project, helping other users and discuss all things MLOps.
...
-   Blog <https://modelserving.com>
+
+.. toctree::
+   :caption: BentoCloud
+   :hidden:
+   
+   bentocloud/getting-started/index
+   bentocloud/how-tos/index
+   bentocloud/topics/index
+   bentocloud/tutorials/index
```
Comment (ssheng) [inline docs/source/index.rst:140]:
```suggestion
```

File: docs/source/bentocloud/getting-started/ship.rst (hunk 6 above)
Comment (ssheng) [inline docs/source/bentocloud/getting-started/ship.rst:76]:
```suggestion
Bento can be deployed in two ways: as an Online Service or as a Serverless Function. The choice depends on your specific use-case. You can learn more about the implications and best scenarios for each method in `:ref:Deploying Bento <concepts/deploy:Deploying Bento>`.
```

File: docs/source/bentocloud/references/index.rst [hunk 28]
```diff
@@ -0,0 +1,6 @@
...
+
+.. TODO:
```
Comment (ssheng) [inline docs/source/bentocloud/references/index.rst:5]:
```suggestion
.. TODO::
```

File: docs/source/bentocloud/getting-started/build.rst [hunk 29]
```diff
@@ -0,0 +1,99 @@
+====================
+Build AI Application
+====================
```
Comment (ssheng) [inline docs/source/bentocloud/getting-started/build.rst:3]:
```suggestion
====================
Build with Rapid Iterations
====================
```

File: docs/source/bentocloud/getting-started/index.rst [hunk 30]
```diff
@@ -0,0 +1,65 @@
...
+.. grid:: 1 3 1 3
+    :gutter: 3
+    :margin: 0
+    :padding: 3 4 0 0
+
+    .. grid-item-card:: 🔨 Build 
+        :link: build
+        :link-type: doc
+
+        Streamline your model creation with BentoML's flexible and integrated development environment.
```
Comment (ssheng) [inline docs/source/bentocloud/getting-started/index.rst:45]:
```suggestion
        Prototype AI applications with rapid iterations with BentoML.
```

File: docs/source/bentocloud/getting-started/index.rst [hunk 31]
```diff
@@ -0,0 +1,65 @@
...
+        :link-type: doc
+
+        Deploy your AI applications effortlessly and efficiently to production.
+
+    .. grid-item-card:: 📈 Scale
+        :link: scale
+        :link-type: doc
+
+        Scaling your work with a fully managed platform that aims to relieve users of hardware concerns, 
+        allowing you to pay only for what you use.
```
Comment (ssheng) [inline docs/source/bentocloud/getting-started/index.rst:58]:
```suggestion
        Scaling your work with a fully managed platform that aims to relieve users of infrastructure concerns, 
        allowing you to pay only for what you use.
```

File: docs/source/bentocloud/getting-started/build.rst [hunk 32]
```diff
@@ -0,0 +1,99 @@
...
+
+As BentoML is built with **Python**, it offers an intuitive coding experience for ML practitioners, seamlessly integrating with popular libraries within the Python ecosystem. This makes your coding experience both familiar and efficient.
+
+BentoML embraces **open standards** for AI applications and promotes **best practices** to enhance the quality of your work. It offers a versatile framework capable of unifying **online**, **offline**, and **streaming** workloads.
+
+In essence, BentoML aims to simplifies the process of building ML models, ensuring your team can focus on what truly matters: creating AI applications that solve real-world problem.
+
+---------------------------
+Building Bento Applications
+---------------------------
```
Comment (ssheng) [inline docs/source/bentocloud/getting-started/build.rst:21]:
```suggestion
---------------------
Building Applications
---------------------
```

File: docs/source/bentocloud/getting-started/build.rst [hunk 33]
```diff
@@ -0,0 +1,99 @@
...
+
+   git clone https://github.com/bentoml/transformers-nlp-service.git
+
+Then, install the required dependencies:
+
+.. code-block:: bash
+
+   pip install -r requirements.txt
+
+To serve your model as an HTTP server, utilize the **`bentoml serve`** CLI command. This starts a local server at `localhost:3000`, making your model accessible as a web service.
```
Comment (ssheng) [inline docs/source/bentocloud/getting-started/build.rst:37]:
```suggestion
To serve your model as an HTTP server, utilize the ``bentoml serve`` CLI command. This starts a local server at `localhost:3000`, making your model accessible as a web service.
```

**Review by ssheng — APPROVED** [review]

File: docs/source/bentocloud/getting-started/ship.rst [hunk 34]
```diff
@@ -0,0 +1,151 @@
...
+
+- **Deploy** your Bento into production within minutes
+- Manage application versioning and deployment revisions
+- Integrate with popular **MLOps** ecosystem tooling
+- Activate **GPU** acceleration with a simple click
+- Utilize out-of-the-box DevOps and ML **Observability**
+
+------------------------
+Deploying to BentoCloud
+------------------------
```
Comment (ssheng) [inline docs/source/bentocloud/getting-started/ship.rst:17]:
```suggestion
```

File: docs/source/bentocloud/getting-started/scale.rst [hunk 35]
```diff
@@ -0,0 +1,70 @@
+================
+Scale for Growth
+================
+
+In the previous parts of our 'Getting Started' series:
+
+- :doc:`Getting Started -- Build AI Application <build>`
+- :doc:`Getting Started -- Ship To Production <ship>`
```
Comment (ssheng) [inline docs/source/bentocloud/getting-started/scale.rst:8]:
```suggestion
- :ref:`<build>`
- :ref:`<ship>`
```

File: docs/source/bentocloud/getting-started/scale.rst [hunk 36]
```diff
@@ -0,0 +1,70 @@
+================
+Scale for Growth
+================
+
+In the previous parts of our 'Getting Started' series:
+
+- :ref:`<build>`
+- :ref:`<ship>`
```
Comment (ssheng) [inline docs/source/bentocloud/getting-started/scale.rst:8]:
```suggestion
* :doc:`/bentocloud/getting-started/build`
* :doc:`/bentocloud/getting-started/ship`
* :doc:`/bentocloud/getting-started/scale`
```

File: docs/source/bentocloud/getting-started/ship.rst [hunk 37]
```diff
@@ -0,0 +1,148 @@
...
+The topics covered in this article are:
+
+- **Deploy** your Bento into production within minutes
+- Manage application versioning and deployment revisions
+- Integrate with popular **MLOps** ecosystem tooling
+- Activate **GPU** acceleration with a simple click
+- Utilize out-of-the-box DevOps and ML **Observability**
+
+
+Having crafted a Bento as illustrated in the preceding section,  :doc:`Getting Started -- Build AI Application <build>`, we now progress to the next stage: launching it into production using BentoCloud.
```
Comment (ssheng) [inline docs/source/bentocloud/getting-started/ship.rst:16]:
```suggestion
Having crafted a Bento as illustrated in the preceding section, :doc:`/bentocloud/getting-started/build`, we now progress to the next stage: launching it into production using BentoCloud.
```

**Review by ssheng — APPROVED** [review]

**Pull request merged**